banzai/images.py

```diff
@@ -125,6 +128,19 @@ def writeto(self, filename, fpack=False):
                 self.filename += '.fz'
             shutil.move(os.path.join(temp_directory, base_filename), filename)
 
+    def regenerate_astropy_data_table_from_fits_table_hdu(self, hdu_list, table_extension_name):
```

Comment:
If this is just for testing, then it should go in `test_image.py`

Reply:
it is for reorganizing tables into proper astropy tables once they have been loaded from disc

Reply:
Actually, it can go outside of the Image class in `image.py`.

Reply:
is this because it is 'static'?

Reply:
Yep